packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -204,7 +205,7 @@ export class WalletConnectConnection implements WalletConnection {
         accountAddress: string,
         type: AccountTransactionType,
         payload: AccountTransactionPayload,
-        parameters?: Record<string, unknown>,
+        parameters?: SmartContractParameters,
```

Comment:
Does the mobile wallet support the new types? @shjortConcordium If not, I think we should only include the parts of the unit type that are supported everywhere in the interface. I assume the browser wallet implementation could extend this to the full type.

Reply:
The mobile should "support" these types, but the parameters are serialized before sending them to the MW.